### ISSUE TITLE
J4 RC5 a11y - increae color contrast for drop down

### ISSUE THREAD

**Issue by coolcat-creations** [joomla/joomla-cms#35010]
### Steps to reproduce the issue
<img width="745" alt="grafik" src="https://user-images.githubusercontent.com/828371/127768018-cc30d2f8-4074-4f7a-be9a-20039e4e6e96.png">
<img width="285" alt="grafik" src="https://user-images.githubusercontent.com/828371/127768027-34f22511-9ffe-4fa1-aeb5-34df3adcd0d9.png">

The color contrast in the green drop downs is not enough

### Expected result
White and a darker green would work
<img width="756" alt="grafik" src="https://user-images.githubusercontent.com/828371/127768070-9a34fc5e-d673-40ad-9673-40e1a0aace2c.png">

### Actual result
Contrast Checker fails

### System information (as much as possible)
Joomla 4 RC5 and Firefox Developer Edition

**Comment by brianteeman** [comment]
I'm a bit confused - which green are you referring to. If it is the green box with the black indicator then you cut the graphic before it shows the relevant part ;)

![image](https://user-images.githubusercontent.com/1296369/127768338-943aa25a-199b-49eb-b8ac-15bafcdfd78c.png)

If its the green text  on the white background then it is also fine

![image](https://user-images.githubusercontent.com/1296369/127768483-6af97475-1da0-4fda-8b55-62b5a219f1b6.png)

**Comment by coolcat-creations** [comment]
I tested the black chevron on the green and It shows both fail in the color contrast checker?

**Comment by chmst** [comment]
The black chevron on dark green indeed has bad contrast.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35010">issues.joomla.org/tracker/joomla-cms/35010</a>.</sub>

**Comment by brianteeman** [comment]
It is not text. It's a user interface component which as shown in the screenshot passes because the contrast only needs to be 3:1

**Comment by coolcat-creations** [comment]
Ok, got you - however the contrast is so weak that it confused even me and I am not color blind. this could be optimized or not?

**Comment by brianteeman** [comment]
The problem is that because of the _strange_ way the arrow has been created that has to stay as black and changing the green is the only option but that has a knock on effect throughout the template.

For me the only real solution is to fix the way the arrow is created

**Comment by coolcat-creations** [comment]
Seems we could change the css to 

`.form-select.form-select-success, .form-select.form-select-danger {
background-image: url("/administrator/templates/atum/images/select-bg-white.svg");
}`

and add a white arrow svg.

I am currently with the Joomla Kids Training and not able to do the PR but hopefully next week if no one has time to do it.

Or someone could fix the strange markup that is done there...

**Comment by Krshivam25** [comment]
I'm working on this issue. Thanks

**Comment by Quy** [comment]
Is it necessary to have the color background? Maybe resurrect this PR https://github.com/joomla/joomla-cms/pull/28755

**Comment by Krshivam25** [comment]
> Is it necessary to have a color background? Maybe resurrect this PR #28755

Maybe not as another tab is having white color background.
![color](https://user-images.githubusercontent.com/49578630/128063177-10e62e14-0e7b-4416-b5ce-b6c3f5897fce.png)
